modular_skyrat/modules/customization/modules/clothing/suits/misc.dm

```diff
@@ -279,13 +279,13 @@
 	icon_state = "kuban_cossak"
 	mutant_variants = NONE
 
-/obj/item/clothing/suit/cossack/sec
+/obj/item/clothing/suit/armor/cossack/sec
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
 	name = "security ukrainian coat"
+	desc = "Cossacks, oh cossacks, let's go, cossachenki."
```

Comment:
No thanks. No politics in item description.

Reply:
thats what the default one is


its literally just a lyric to a song about the cossacks

Reply:
ccing in from discord for wider context:

"it isnt a political group
its an ethnic group
its a cultural group
and, its quite literally a cossack coat
cultural/ethnic group
it'd be like saying a reggi hat is political"